ERCS/erc-7996.md

```diff
@@ -0,0 +1,78 @@
+---
+eip: 7996
+title: Contract Feature Detection
+description: Method to publish and detect contract features that lack an ERC-165 interface
+author: raffy.eth (@adraffy)
+discussions-to: https://ethereum-magicians.org/t/erc-7996-contract-feature-detection/24975
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-07-07
+requires: 165
+---
+
+## Abstract
+
+Creates a standard method to publish and detect what features a smart contract implements that lack an [ERC-165](./eip-165.md) interface.
```

Comment:
Your abstract is lacking a bit of technical meat. You should describe, at a high level, what the "standard method" actually is.